src/seq_mv/csr_spgemm_device.h

```diff
@@ -48,27 +58,41 @@ constexpr HYPRE_Int T_GROUP_SIZE[11]    = { 0,  2,  4,   8,  16,  32,   64,  128
  * these are under the assumptions made in spgemm on block sizes: only use in csr_spgemm routines
  * where we assume CUDA block is 3D and blockDim.x * blockDim.y = GROUP_SIZE which is a multiple
  * of HYPRE_WARP_SIZE
```

Comment:
My fault, we should have updated this comment: GROUP_SIZE is not necessarily a multiple of WARP_SIZE anymore. Maybe just delete "which is ...".

Reply:
I went ahead and fixed this comment.